ng2-components/ng2-alfresco-documentlist/src/components/document-list.spec.ts

```diff
@@ -158,69 +150,46 @@ describe('DocumentList', () => {
-        let path = '/';
-
         let node = new FolderNode('<display name>');
 
-        spyOn(documentList, 'getNodePath').and.returnValue(path);
-        spyOn(documentList, 'loadFolderByPath').and.returnValue(Promise.resolve(true));
+        spyOn(documentList, 'loadFolder').and.returnValue(Promise.resolve(true));
 
         documentList.navigationMode = DocumentList.SINGLE_CLICK_NAVIGATION;
         documentList.onNodeClick(node);
 
-        expect(documentList.currentFolderPath).toBe(path);
+        expect(documentList.loadFolder).toHaveBeenCalled();
     });
 
     it('should not display folder content when no target node provided', () => {
         expect(documentList.navigate).toBe(true);
-        spyOn(documentList, 'loadFolderByPath').and.stub();
+        spyOn(documentList, 'loadFolder').and.stub();
 
         documentList.onNodeClick(null);
-        expect(documentList.loadFolderByPath).not.toHaveBeenCalled();
+        expect(documentList.loadFolder).not.toHaveBeenCalled();
 
     });
 
     it('should display folder content only on folder node click', () => {
         expect(documentList.navigate).toBe(true);
-        spyOn(documentList, 'loadFolderByPath').and.stub();
+        spyOn(documentList, 'loadFolder').and.stub();
 
         let node = new FileNode();
         documentList.onNodeClick(node);
 
-        expect(documentList.loadFolderByPath).not.toHaveBeenCalled();
+        expect(documentList.loadFolder).not.toHaveBeenCalled();
     });
 
     it('should not display folder content on click when navigation is off', () => {
-        spyOn(documentList, 'loadFolderByPath').and.stub();
+        spyOn(documentList, 'loadFolder').and.stub();
 
         let node = new FolderNode('<display name>');
         documentList.navigate = false;
         documentList.onNodeClick(node);
 
-        expect(documentList.loadFolderByPath).not.toHaveBeenCalled();
-    });
-
-    it('should require node to get path', () => {
-        expect(documentList.getNodePath(null)).toBe(null);
-    });
-
-    it('should display folder content for new folder path', () => {
-        spyOn(documentList, 'loadFolderByPath').and.returnValue(Promise.resolve());
-        let newPath = '/some/new/path';
-        documentList.currentFolderPath = newPath;
-        documentList.ngOnChanges({currentFolderPath: new SimpleChange(null, newPath)});
-        expect(documentList.loadFolderByPath).toHaveBeenCalledWith(newPath);
-    });
-
-    it('should reset to default path', () => {
-        spyOn(documentList, 'loadFolderByPath').and.returnValue(Promise.resolve());
-        documentList.currentFolderPath = null;
-        documentList.ngOnChanges({currentFolderPath: new SimpleChange('', null)});
-
-        expect(documentList.currentFolderPath).toBe(documentList.DEFAULT_FOLDER_PATH);
-        expect(documentList.loadFolderByPath).toHaveBeenCalledWith(documentList.DEFAULT_FOLDER_PATH);
+        expect(documentList.loadFolder).not.toHaveBeenCalled();
     });
 
+    /*
```

Comment:
Remove commented out test or replace `it()` with `xit()` if there is a reason to keep it for now?